ElrondNetwork/elrond-go
# En 7527 index accounts

This PR adds 2 functionalities:
1. ability to index only the given indexes. Before this, only for `transactions` there was the option to index or not. Now, all indexes can be configured.
2. the `accounts` index that will contain data about used accounts (from genesis and from transactions). 

Testing procedure:
1. run a testnet with some elastic observers
2. send some transactions
3. check the accounts index: GET on `elasticURL/accounts/_search` where you should see entries for each used account (genesis + from transactions). The format of an account looks like this:
``` JSON
 {
        "_index": "accounts",
        "_type": "_doc",
        "_id": "erd19j8u0kstdgwmgaqm23dh8lj0g5vxm6skg80yrp237rxwjneyz3qqqjmglw",
        "_score": null,
        "_source": {
          "nonce": 0,
          "balance": "2220555555555555555555560",
          "balanceNum": 2220555.5555555555
        }
}
```
4. additional testing: check that the shown balance is the same balance returned by `/address/` endpoint for a few samples.
